docs/operator/VictoriaMetrics-Operator.md

```diff
@@ -0,0 +1,110 @@
+[![Slack](https://img.shields.io/badge/join%20slack-%23victoriametrics-brightgreen.svg)](http://slack.victoriametrics.com/)
+[![GitHub license](https://img.shields.io/github/license/VictoriaMetrics/operator.svg)](https://github.com/VictoriaMetrics/operator/blob/master/LICENSE)
+[![Go Report](https://goreportcard.com/badge/github.com/VictoriaMetrics/operator)](https://goreportcard.com/report/github.com/VictoriaMetrics/operator)
+[![Build Status](https://github.com/VictoriaMetrics/VictoriaMetrics/workflows/main/badge.svg)](https://github.com/VictoriaMetrics/operator/actions)
+
+![Victoria Metrics logo](logo.png "Victoria Metrics")
+
+# VictoriaMetrics operator
+
+## Overview
+
+ Design and implementation inspired by [prometheus-operator](https://github.com/prometheus-operator/prometheus-operator). It's great a tool for managing monitoring configuration of your applications. VictoriaMetrics operator has api capability with it.
+So you can use familiar CRD objects: `ServiceMonitor`, `PodMonitor`, `PrometheusRule` and `Probe`. Or you can use VictoriaMetrics CRDs:
+- `VMServiceScrape` - defines scraping metrics configuration from pods backed by services.
+- `VMPodScrape` - defines scraping metrics configuration from pods.
+- `VMRule` - defines alerting or recording rules.
+- `VMProbe` - defines a probing configuration for targets with blackbox exporter.
+
+Besides, operator allows you to manage VictoriaMetrics applications inside kubernetes cluster and simplifies this process [quick-start](/Operator/quick-start.html) 
+With CRD (Custom Resource Definition) you can define application configuration and apply it to your cluster [crd-objects](/Operator/api.html). 
+
+ Operator simplifies VictoriaMetrics cluster installation, upgrading and managing.
+ 
+ It has integration with VictoriaMetrics `vmbackupmanager` - advanced tools for making backups. Check backup [docs](/Operator/backups.html)
+
+## Use cases
+
+ For kubernetes-cluster administrators, it simplifies installation, configuration, management for `VictoriaMetrics` application. And the main feature of operator -  is ability to delegate applications monitoring configuration to the end-users.
+ 
+ For applications developers, its great possibility for managing observability of applications. You can define metrics scraping and alerting configuration for your application and manage it with an application deployment process. Just define app_deployment.yaml, app_vmpodscrape.yaml and app_vmrule.yaml. That's it, you can apply it to a kubernetes cluster. Check [quick-start](/Operator/quick-start.html) for an example.
+
+## Operator vs helm-chart
+
+VictoriaMetrics provides [helm charts](https://github.com/VictoriaMetrics/helm-charts). Operator makes the same, simplifies it and provides advanced features.
+
+## Documentation
+
+- quick start [doc](quick-start.html)
```

Comment:
hm, not sure if it needed, because at docs site we have side bar navigation scroll.

Reply:
So, removing it?

Reply:
yes